examples/AspNetCore/OData/ODataBasicExample/Program.cs

```diff
@@ -4,7 +4,8 @@
 
 // Add services to the container.
 
-builder.Services.AddControllers().AddOData();
+builder.Services.AddControllers()
```

Comment:
Did this auto-wrap? I'd prefer that this be one of the following:

1. A single line
2. `.AddControllers` and `.AddOData` align vertically

I don't have a super strong opinion about which one you choose

Reply:
Fixed